### ISSUE TITLE
Key expiry notifications every second.

### ISSUE THREAD

**Issue by darshinimashar** [tailscale/tailscale#1125]
Key expiry notifications on windows, keep sending notifications every second as soon as we close it.

Please watch the video for more clarification,
https://share.getcloudapp.com/2Nu0ZjOR

**Comment by sean-tock** [comment]
Hi! I'm the one who recorded the video, and happy to answer any additional questions.

Some more details:

* I'm on client version 1.2.4 on macOS Big Sur.
* I recorded the video of the issue on Jan 12, 2020. To the best of my knowledge, I was on 1.2.4 on that date.
* Even after I stopped the recording, the notifications kept coming I received dozens of notifications (likely over a hundred) over the course of ~5 minutes until I re-authenticated.
* This has been happening every time I need to re-authenticate for as long as I can remember, including on my previous laptop on a lower version of macOS.
* 5 members of my team (myself included) expressed on Jan 11, 2020 that they've been experiencing this issue, so I know it's not isolated to me.
